contrib/travis.sh

```diff
@@ -16,15 +16,30 @@ function edit_readme {
     sed -i "s/$PENULTIMATE_TAG/$TRAVIS_TAG/g" README.md
 }
 
-function commit_changed_readme {
+function setup_git {
     git config --global user.email "buils@travis-ci.com"
     git config --global user.name "Travis CI"
-    git add README.md
-    git commit -s -m "Bump README with the new release tag: $TRAVIS_TAG"
+}
+
+function commit_and_push {
+    git commit -s -m "$@"
     git pull origin master --rebase
     git push https://"$GITHUB_TOKEN"@github.com/ceph/cn master
 }
 
+function commit_specfile {
+    pushd contrib
```

Comment:
indent of 4 and then indent of 2?

Reply:
From my editor, this line (31) is matching the rest of the code identation's style

Reply:
Yes, but you next indent is 2 where it should be 4 :)

Reply:
ok. fixed.